### PR TITLE
Exercise 2

### PR DESCRIPTION
**Pull request by noratop**

### CODE REVIEW

**Comment by ziad-saab** [comment]
:+1:

File: index.html [hunk 1]
```diff
@@ -9,6 +9,32 @@
   </head>
   
   <body>
-    <!-- write your html here-->
-  </body>
+<aside class="dialog-box">
```
Comment (nyanofthemoon) [inline index.html]: @noratop Please fix identation.

Comment (noratop) [inline index.html]: @nyanofthemoon indentation fixed

Comment (nyanofthemoon) [inline index.html]: @noratop Did you commit the modifications?

Comment (noratop) [inline index.html]: oh indeed my commit is not here

**Comment by noratop** [comment]
@nyanofthemoon here is the indentation fix

**Comment by nyanofthemoon** [comment]
@noratop Thank you- it looks really clean :dancer:

**Pull request closed without merging**